src/assets/i18n/fr.json5

```diff
@@ -5510,6 +5564,27 @@
   // "search.filters.search.submit": "Submit",
   "search.filters.search.submit": "Soumettre",
 
+  //"search.filters.operator.equals.text": "Equals",
+  "search.filters.operator.equals.text": "Est égal à",
+
+  //"search.filters.operator.notequals.text": "Not Equals",
+  search.filters.operator.notequals.text": "N'est pas égal à",
```

Comment:
This line is missing a quote (") at the very beginning

Reply:
Done